content/docs/command-reference/exp/init.md

```diff
@@ -0,0 +1,137 @@
+# exp init
+
+Initializes experiments.
+
+## Synopsis
+
+```usage
+usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
+                    [--explicit] [--name NAME] [--code CODE]
+                    [--data DATA] [--models MODELS] [--params PARAMS]
+                    [--metrics METRICS] [--plots PLOTS] [--live LIVE]
+                    [--type {default,live}]
+                    [command]
+```
+
+## Description
+
+`dvc exp init` helps you quickly get started with experiments, with the command
+to run and tracks data, params, source code, models, metrics and plots by
+default.
+
+It reduces boilerplate for initializing [pipeline](/doc/command-reference/dag)
+stages in a `dvc.yaml` file by assuming sane defaults about the location of your
+data, parameters, source code, models, metrics and plots, which can be
+customized through config.
+
+It also offers guided `--interactive` mode for creating a stage to be `exp run`
+later. `dvc exp init` supports creating different types of stages, eg: live if
+you are using dvclive to track metrics and plots, etc.
+
+### The `command` argument
+
+The `command` argument is optional, if you are using `--interactive` mode. The
+`command` sent to `dvc exp init` can be anything your terminal would accept and
```

Comment:
Should we put this under **Options** like for `targets` in https://dvc.org/doc/command-reference/repro#options (and other places I think) ? That was an initiative of @skshetry actually 🙂

Reply:
Let's keep it as it is for now, as we also need to do something for `stage add`/`run`?

Reply:
Why keep it as a section here when we moved it in `repro` ? We had lots of good reasons (which you proposed), what's different?

> do something for stage add/run

Out of scope but we can create an issue.

Reply:
Because it's better to do it at the same time for all commands. I am going with what we have right now, let's handle it later?

Reply:
p.s. we don't always make changes for all commands at once, we can take them as they come. BTW I actually like a section better than having it under Options, I'm just confused since we discussed this a lot for `repro` and you (and Ivan) were very sure about putting it under Options 😆

Reply:
> Because it's better to do it at the same time for all commands. I am going with what we have right now, let's handle it later?

https://github.com/iterative/dvc.org/pull/3071#pullrequestreview-829161122